srcpkgs/python3-Pyro4/template

```diff
@@ -1,20 +1,31 @@
 # Template file for 'python3-Pyro4'
 pkgname=python3-Pyro4
-version=4.80
-revision=4
+version=4.81
+revision=1
 wrksrc="${pkgname#*-}-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-serpent"
+# checkdepends="python3-serpent python3-msgpack python3-dill python3-cloudpickle"
+checkdepends="python3-serpent"
 short_desc="Distributed Object Middleware (IPC/RPC) for Python3"
 maintainer="pulux <pulux@pf4sh.de>"
 license="MIT"
 homepage="https://pythonhosted.org/Pyro4/"
 distfiles="${PYPI_SITE}/P/Pyro4/Pyro4-${version}.tar.gz"
-checksum=46847ca703de3f483fbd0b2d22622f36eff03e6ef7ec7704d4ecaa3964cb2220
+checksum=e130da06478b813173b959f7013d134865e07fbf58cc5f1a2598f99479cdac5f
 
 conflicts="python-Pyro4>=0"
 
+do_check() {
+	# Tests depend on python3-dill and python3-cloudpickle, which aren't packaged
+	rm -f tests/PyroTests/test_daemon.py tests/PyroTests/test_serialize.py
+	# Tests can't get addr
+	rm -f tests/PyroTests/test_naming.py tests/PyroTests/test_naming2.py
+	rm -f tests/PyroTests/test_socket.py tests/PyroTests/test_util.py
+	# PYTHONPATH=src python3 -E -Wall -tt -bb tests/run_testsuite.py
```

Comment:
`do_check` isn't actually doing anything after remove those that don't work.

Reply:
yes I know, but the run_testsuite.py depends on Pyro4 and this was not found in the PR-Test Env. How can we handle this?

Reply:
If getting tests to work without installing the package, I would delete `checkdepends` and the `do_check` altogether, set `make_check="no"` and leave a comment to the effect that tests don't work unless the package is installed (cf. `python3-numpy`).